components/migrate/steps/SignLoader1.vue

```diff
@@ -211,4 +213,12 @@ const whichIcon = () => {
 
   return iconIdle
 }
+
+const textStatus = computed(() => {
+  if (whichIcon().icon === 'circle') {
+    return 'text-k-grey'
+  }
```

Comment:
It's a bit confusing here. It should be controlled by the current status, instead of the type of the icon. 

How about this?
```
export const iconIdle = {
  icon: 'circle',
  class: 'has-text-grey',
  text-color: xxxx
}
```

Reply:
good idea 👍🏻